src/arrayMethodSort.js

```diff
@@ -4,8 +4,34 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
You can create compareFn  outside of applyCustomSort() function and set it as default.
It will look like this.
<img width="631" alt="317202373-90408886-606c-4da6-9b68-915c0ea6c6ae" src="https://github.com/mate-academy/js_array-method-sort/assets/159737539/65d42599-2810-4fa9-aab4-ce260336ab90">